apps/mediator/src/events.ts

```diff
@@ -0,0 +1,12 @@
+import { BaseEvent } from '@credo-ts/core'
+
+export enum MediatorEventTypes {
+  DidCommMessageQueued = 'DidCommMessageQueued',
+}
+
+export interface DidcommMessageQueuedEvent extends BaseEvent {
```

Comment:
I think we can find a way to homogenize these event types. In the Postgres implementation we are adding a few more things like the message (to include its id on the Push notification, as stated in FCM/APNs push notification RFCs) and session object (to determine whether we should send or not the Push notification, see
https://github.com/2060-io/didcomm-mediator/blob/ca0d3ced6520145e2e5525ccdab3aa3cc19d2546/src/agent/initDidCommMediatorAgent.ts#L137).

Maybe we are sending a bit too much but if we can define a common event type and add it into Credo's main DIDComm module for 0.6 release it would be great.

Reply:
Yeah i just added the minimal amount possible, but we can extend it with the needed parameters for the postgres implementation as well